### ISSUE TITLE
[accessibility] Controls' icons should be explicitly hidden from assistive technology

### ISSUE THREAD

**Issue by Malvoz** [openlayers/openlayers#12041]
Screen readers currently announce the "icons" or characters inside of controls because they're not hidden from <span title="Assistive Technologies">ATs</span>.

Taking the zoom in control as an example, a screen reader such as [NVDA](https://www.nvaccess.org/) will announce <q>Plus button zoom in</q> on focus, and <q>Plus</q> on hover.

![openlayers-controls-accessible-name](https://user-images.githubusercontent.com/26493779/108210918-66ac4e00-712c-11eb-9788-277deb0ed729.png)

Similarly, the attribution control is announced as <q>i button attributions</q> on focus, and <q>i</q> on hover, because the character "i" (for "info") is not hidden from screen readers.

I propose that these icons/characters be wrapped in a child `<span aria-hidden="true">` as a child element of `<button>` to prevent screen readers from announcing them.

(Ideally this should also be a consideration for custom controls, if only by [updating the examples](https://github.com/openlayers/openlayers/issues/12042).)

**Comment by walkermatt** [comment]
@Malvoz my understanding is that the [`ol/control/Zoom~Zoom`](https://openlayers.org/en/latest/apidoc/module-ol_control_Zoom.html) `zoomInLabel` and `zoomOutLabel` properties facilitate wrapping the icons/characters in a child `<span aria-hidden="true">`.

The following takes a different approach of instead adding a span with a descriptive `aria-label` which I think results in ATs describing the zoom buttons appropriately:

```js
import Map from 'ol/Map.js';
import OSM from 'ol/source/OSM.js';
import TileLayer from 'ol/layer/Tile.js';
import View from 'ol/View.js';
import {defaults} from 'ol/control/defaults';

// Customise the Zoom control to add a aria-label attribute to each button
const zoomInLabel = document.createElement('span');
zoomInLabel.ariaLabel = 'Zoom in';
zoomInLabel.append('+');

const zoomOutLabel = document.createElement('span');
zoomOutLabel.ariaLabel = 'Zoom out';
zoomOutLabel.append('-');

const controls = defaults({
  zoomOptions: {
    zoomInLabel: zoomInLabel,
    zoomOutLabel: zoomOutLabel
  }
});

// Create a Map instance, specifying our custom control options via
// the `controls` option
const map = new Map({
  layers: [
    new TileLayer({
      source: new OSM(),
    }),
  ],
  target: 'map',
  view: new View({
    center: [0, 0],
    zoom: 2,
  }),
  controls: controls
});
```

I'd welcome your thoughts.
